src/galileo/logger/logger.py

```diff
@@ -1198,17 +1183,59 @@ def start_session(
                 self._logger.error("Failed to search for session with external ID %s", external_id, exc_info=True)
 
         self._logger.info("Starting a new session...")
-        session = async_run(
-            self._client.create_session(
-                SessionCreateRequest(name=name, previous_session_id=previous_session_id, external_id=external_id)
-            )
+
+        session = await self._client.create_session(
+            SessionCreateRequest(name=name, previous_session_id=previous_session_id, external_id=external_id)
         )
 
         self._logger.info("Session started with ID: %s", session["id"])
-
         self.session_id = str(session["id"])
         return self.session_id
 
+    @nop_async
+    async def async_start_session(
```

Comment:
In protect we use `ainvoke` for the async version of `invoke` - should we follow the same naming pattern here?